developer/language/reference/if.php

```diff
@@ -73,6 +73,7 @@
 
 <p>The <code>if()</code> statement was introduced in Keyman 8.0.</p>
 <p>Keyman 9.0: Added support for variable system stores.</p>
+<p>Keyman for macOS 11.0.221 added support for `if()`, `set()`, `save()` and `reset()` for variable stores.</p>
```

Comment:
```suggestion
<p>Keyman for macOS 11.0.221 added support for <code>if()</code>, <code>set()</code>, <code>save()</code> and <code>reset()</code> for variable stores.</p>
```

I don't think the site styles Markdown ticks, so shouldn't we stick with `<code>` tags?

Reply:
Ugh, this is why I shouldn't work while jet lagged! :) Good catch!